src-django/api/tests/test_procedures.py

```diff
@@ -9,28 +9,48 @@
 
 
 class ProcedureTest(TestCase):
+
     def setUp(self):
         self.client = Client()
         self.token = Token.objects.get(user=factories.UserFactory())
-        self.procedure_url = '/api/procedures/'
+        self.procedure_url = '/api/procedures'
         self.data = {
             "title": "Example Title",
             "author": "An Author"
         }
+        self.user = factories.UserFactory()
+        initialize_permissions
 
-    @initialize_permissions
     def test_created_procedure_has_correct_owner(self):
-        user = factories.UserFactory()
-
         response = self.client.post(
             path=self.procedure_url,
             data=json.dumps(self.data),
             content_type='application/json',
-            HTTP_AUTHORIZATION=add_token_to_header(user, self.token)
+            HTTP_AUTHORIZATION=add_token_to_header(self, self.user, self.token)
         )
 
-        assert_equals(response.status_code, status.HTTP_200_OK)
+        assert_equals(response.status_code, status.HTTP_201_CREATED)
         body = json.loads(response.content)
 
         assert_true('owner' in body)
-        assert_equals(body['owner'], user.id)
+        assert_equals(body['owner'], self.user.id)
+
+    def test_procedure_comes_with_pages(self):
+        procedure = factories.ProcedureFactory(owner=self.user)
+        factories.PageFactory(procedure=procedure)
+        flags = {
+            "only_return_id": "false"
```

Comment:
Previous tests also used double quotes for the data passed into the request, so I'll leave it.